qutebrowser/misc/editor.py

```diff
@@ -68,29 +68,26 @@ def on_proc_closed(self, exitcode, exitstatus):
         Callback for QProcess when the editor was closed.
         """
         log.procs.debug("Editor closed")
-        if exitstatus != QProcess.NormalExit:
-            # No error/cleanup here, since we already handle this in
-            # on_proc_error.
+        if exitstatus != QProcess.NormalExit or exitcode != 0:
+            message.error("Editor processed failed to exit cleanly."
```

Comment:
Or even better, `"Editor process exit with status {}."` A bit more feedback is always welcome.